linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/ansible/shared.yml

```
@@ -8,7 +8,7 @@
   shell: |
     set -o pipefail
     find / -not \( -fstype afs -o -fstype ceph -o -fstype cifs -o -fstype smb3 -o -fstype smbfs -o -fstype sshfs -o -fstype ncpfs -o -fstype ncp -o -fstype nfs -o -fstype nfs4 -o -fstype gfs -o -fstype gfs2 -o -fstype glusterfs -o -fstype gpfs -o -fstype pvfs2 -o -fstype ocfs2 -o -fstype lustre -o -fstype davfs -o -fstype fuse.sshfs \) -type f \( -perm -4000 -o -perm -2000 \) 2> /dev/null
-    executable: /bin/bash
+  executable: /bin/bash
```

This syntax seems to be invalid in this case.
According to the documentation, you can't do this when you are passing the shell command in "free form" way to the shell module.
I tested this approach and it seems to work.
```suggestion
  args:
    executable: /bin/bash
```

---

Thanks for reviewing @vojtapolasek . I saw this after sending the PR but was too late to update. : ))
I am even considering to remove this parameter. I will confirm and fix soon.

---

Ok, we can't remove the `executable: /bin/bash` parameter since the command is using non-posix features (redirection).